Sources/PackageCollections/PackageCollections+Validation.swift

```diff
@@ -8,8 +8,120 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
+import PackageModel
 import TSCBasic
 
+extension JSONPackageCollectionModel.V1 {
+    public struct Validator {
```

Comment:
Could also more the `Validator` to its own file but this seemed an appropriate place.

The main point is to move it out of the model files, because it requires `PackageModel` and `TSCUtility`.